web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -178,10 +179,10 @@ export function useFilterSearch() {
         });
       };
       const getResourceType = () => {
-        let resourceTypes = [
-          'servers' as const,
-          'databases' as const,
-          'kubes' as const,
+        let resourceTypes: SupportedResourceType[] = [
+          'node' as const,
+          'db' as const,
+          'kube_cluster' as const,
```

Comment:
`const` is not needed AFAIR if you declared the type of the variable itself.

Reply:
They were needed here because the type was specified for an array that was then filtered and TS couldn't narrow down the type:
```
Type  string  is not assignable to type  SupportedResourceType 
```
I overcome this by extracting a const var:
```ts
const SUPPORTED_RESOURCE_TYPES: SupportedResourceType[] = [
  'node',
  'db',
  'kube_cluster',
];
```